run-build-functions.sh

```diff
@@ -424,7 +427,7 @@ install_dependencies() {
     swiftenv rehash
     echo "Finished restoring cached Swift version"
   fi
-  
+
```

Comment:
This whitespace keeps being prettified in my IDE, I think it's probably using a tab instead of spaces.

Reply:
Looks to me like there was some trailing whitespace on this line that your editor cleaned up. Good with me!